### ISSUE TITLE
Bullet cycling breaks for users of VimWiki syntax

### ISSUE THREAD

**Issue by brennen** [vimwiki/vimwiki#911]
From reports on the PR, I suspect that #907 is breaking for people using regular VimWiki syntax instead of Markdown.

```
Error detected while processing function <SNR>157_CR[7]..vimwiki#lst#kbd_cr[11]..<SNR>170_cr_on_empty_list_item[39]..<SNR>170
_change_level[45]..<SNR>170_indent_line_by[6]..<SNR>170_decrease_level[17]..<SNR>170_indent_cycle_bullets[1]..vimwiki#vars#ge
t_syntaxlocal:                                                                                                               
line   10:                                                                                                                   
E716: Key not present in Dictionary: cycle_bullets
```

To reproduce: Make an empty list while using VimWiki syntax, do something that should change list indentation (hit enter on an empty list item, hit Ctrl-t while in insert mode to indent, etc.)

I checked latest version of the dev branch against my regular wiki and a test one in Markdown.  No issues on the Markdown one.

cc: @robertgzr.

**Comment by bewuethr** [comment]
It breaks for me (I commented on #907), and I'm using markdown.

**Comment by tinmarino** [comment]
@brennen is that fixed 43951e58511 ?
Thank you both for the report. Many tests are missing: Vader is really hard to master.

**Comment by brennen** [comment]
> @brennen is that fixed 43951e5 ?

Tested and working - thanks!
